cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -44,7 +44,7 @@ realHostname=`hostname`
 ATDM_KNOWN_SYSTEM_NAMES_LIST=(
   shiller
   ride
-  mutrino   # Will be repalced by 'cts1'
+  mutrino   # Will be repalced by 'ats1'
```

Comment:
spelling: "replaced"

Reply:
Thanks!  It will be "replaced" when 'ats1' replaces 'mutrino' :-)